index.html

```diff
@@ -26,7 +26,7 @@
         window.onTryOn = (colorwaySizeAssetSKU) => {
             console.log("try on", colorwaySizeAssetSKU)
         }
-        window.afterSignIn = (user, colorwaySizeAssetSKU) => {
+        window.whenSignedIn = (user, colorwaySizeAssetSKU) => {
```

Comment:
mostly to keep it consistent with the other events that start with `when`
```
    whenTryOnReady(colorwaySizeAssetSKU: string, frames: types.TryOnFrames): void;

    whenTryOnLoading(colorwaySizeAssetSKU: string): void;

    whenTryOnFailed(colorwaySizeAssetSKU: string, error: Error): void;

    whenError(colorwaySizeAssetSKU: string, error:UIError): void;
    whenSignedIn(user: types.FirebaseUser, colorwaySizeAssetSKU: string): void;
    whenSignedOut(colorwaySizeAssetSKU: string): void;
```
Should i revert the prefix and use after for all of them?

Reply:
My experience with hooks/callbacks like this is that they generally start with a word that precisely defines the timing. Such as in react "componentWillMount" or "componentDidMount" or in iOS "viewWIllAppear" "viewDidAppear". View and Did being the magic works. Or in database hooks land, its usually afterInsert or beforeInsert etc. I think that precision is important for the users of the SDK.

Reply:
In my opinion, this is not the best moment to change or upgrade naming conventions. We could create a specific task for that later on, and focus this PR on one change only (the spinner feature)

Reply:
can i leave it as is since they are all consistent, and we make another PR to address it later

Reply:
That works. As long as they are where we want them before we integrate with any partners. We'll want to do our best to avoid major changes after the first integration.